### ISSUE TITLE
Source file upload option?

### ISSUE THREAD

**Issue by makhan** [lightoj-dev/bugs-and-features#17]
I am used to working on a problem offline in my local programming environment and submitting the file once the solution is ready. It would be nice to have an option to upload the source code instead of copy pasting into the code editor panel.

**Comment by faiyaz26** [comment]
Implemented, will be released soon.

**Comment by faiyaz26** [comment]
Deployed.
